File: HCI-Project/Backend/Endpoints/Rezervacije/ProvjeriRezervaciju/ProvjeriRezervacijuEndpointRes.cs
using Backend.Data.Modeli;

namespace Backend.Endpoints.Rezervacije.ProvjeriRezervaciju;

public class ProvjeriRezervacijuEndpointRes : BaseResponse
{
    public ZauzetaSoba? DetaljiRezervacije { get; set; }
    public Soba? Soba { get; set; }
    public SobaAranzman? SobaAranzman { get; set; }
}

File: Frontend/src/app/endpoints/rezervacije-endpoint/provjeri-rezervaciju/provjeri-rezervaciju-endpoint-res.ts
import {SobaModel} from "../../../models/sobaModel";
import {AranzmanSobaModel} from "../../../models/aranzmanSobaModel";
import {ZauzetaSobaModel} from "../../../models/zauzetaSobaModel";
import {BaseResponse} from "../../base-response";

export interface ProvjeriRezervacijuEndpointRes extends BaseResponse {
  detaljiRezervacije: ZauzetaSobaModel
  soba: SobaModel
  sobaAranzman: AranzmanSobaModel
}
